src/components/shared/Footer.jsx

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable @next/next/no-img-element */
+
+import Image from "next/image";
+export function Footer() {
+  return (
+    <footer className="footer py-8 relative flex flex-col w-full h-max-[500px] bg-bottom bg-cover md:py-5">
+      <div className="w-full flex justify-center z-20">
+        <div className="relative w-[200px] h-[55px]">
+          <Image src="/dxberience_logo.svg" alt="logo" fill />
+        </div>
+      </div>
+
+      <div className="absolute inset-0 bg-[#171010] bg-opacity-90 z-10" />
+      <div className="relative z-20 text-white">
+        <div className="px-8 md:flex justify-between">
+          <h1 className="py-3 text-2xl font-bold font-IvyPresto md:hidden">
+            Find Your Adventure
+          </h1>
+
+          <h1 className="hidden py-2 text-2xl font-bold font-IvyPresto md:block">
+            No need to think, just experience
+          </h1>
+
+          <ul className="mb-10 text-xs text-[#FFFFFF] font-extralight md:flex gap-5 ">
+            <li className="mt-4">EXPERIENCES</li>
+            <li className="mt-4">CONTACT</li>
+          </ul>
+        </div>
+
+        <div className="w-full h-[0.1px] bg-white bg-opacity-20" />
+
+        <div className="flex justify-evenly items-center text-sm ">
```

Comment:
On desktop give it a justify-center and put a gap

Reply:
@Harithmetic1 

Change has been made to the footer,  a couple of things might not be exactly the same. I think there is some issue with the font. But we can review that later I think It looks good at the moment.